src/dataflow/download.ts

```diff
@@ -10,26 +10,33 @@ export default async function downloadFile(
   type: DownloadMediaType,
   token?: string
 ) {
-  const response = await fetchWithHeaders(url, token);
-  if (response.ok) {
-    return downloadBlob(response, title, type);
+  let response;
+
+  try {
+    response = await fetchWithHeaders(url, token);
+  } catch (err) {
+    // There was a network or CORS error. In case this was a CORS error, retry the request without
+    // the X-Requested-With header. Some distributors' servers may not have this header whitelisted
+    // for CORS.
```

Comment:
Any thoughts on how to surface this so that we know to ask a distributor to fix?